frontends/web/src/routes/device/waiting.tsx

```diff
@@ -27,6 +32,15 @@ export const Waiting = () => {
   const { t } = useTranslation();
   const { isDarkMode } = useDarkmode();
 
+  const keystores = useKeystores();
+  const devices = useSync(getDeviceList, syncDeviceList);
+  const hasAccounts = (keystores !== undefined && keystores.length) || (devices !== undefined && Object.keys(devices).length);
```

Comment:
the name of this var is confusing I think. In this moment there are no accounts :P 

how about `loading = ...`?

Reply:
> In this moment there are no accounts

Ah fair enough - I thought because keystore & devices are truthy it means we have an account, but not exactly at all (especially we're actually loading for accounts here).

Thanks.